src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -76,10 +76,15 @@ const Query: FC<PipeProp> = ({Context}) => {
   const injectIntoEditor = useCallback(
     (fn): void => {
       let text = ''
-      if (fn.name === 'from' || fn.name === 'union') {
-        text = `${fn.example}`
+      if (isFlagEnabled('fluxDynamicDocs')) {
+        // only fluxTypes with <- sign require a pipe forward sign
+        text = fn.fluxType[1] === '<' ? `  |> ${fn.example}` : `${fn.example}`
```

Comment:
you might consider something like:

```ts
fn.fluxType.includes('<-') ? ` |> ${fn.example}` : `${fn.example}`
```
Seems slightly less brittle and more explicit in the business rule (it will only work with `<-`, rather than `<afasdfasdf` (which may not be an issue). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes.

Can also consider [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) if the `<-` sequence needs to be at the beginning of the string

Reply:
but I don't see any problems with using `startsWith`. I'll update the code with that. Thanks bucky :)